game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -148,146 +148,116 @@ private void createAndSaveStats(final boolean showPhaseStats) {
       return;
     }
-    final StringBuilder text = new StringBuilder(1000);
-    try {
+    try (Writer writer = Files.newBufferedWriter(chooser.getSelectedFile().toPath(), StandardCharsets.UTF_8)) {
       gameData.acquireReadLock();
       final GameData clone = GameDataUtils.cloneGameData(gameData);
-      final IStat[] stats = statPanel.getStats();
-      // extended stats covers stuff that doesn't show up in the game stats menu bar, like custom resources or tech
-      // tokens or # techs, etc.
-      final IStat[] statsExtended = statPanel.getStatsExtended(gameData);
-      final String[] alliances = statPanel.getAlliances().toArray(new String[0]);
-      final PlayerId[] players = statPanel.getPlayers().toArray(new PlayerId[0]);
-      // its important here to translate the player objects into our game data
-      // the players for the stat panel are only relevant with respect to the game data they belong to
-      Arrays.setAll(players, i -> clone.getPlayerList().getPlayerId(players[i].getName()));
-      text.append(defaultFileName).append(",");
-      text.append("\n");
-      text.append("TripleA Engine Version: ,");
-      text.append(ClientContext.engineVersion()).append(",");
-      text.append("\n");
-      text.append("Game Name: ,");
-      text.append(gameData.getGameName()).append(",");
-      text.append("\n");
-      text.append("Game Version: ,");
-      text.append(gameData.getGameVersion()).append(",");
-      text.append("\n");
-      text.append("\n");
-      text.append("Current Round: ,");
-      text.append(currentRound).append(",");
-      text.append("\n");
-      text.append("Number of Players: ,");
-      text.append(statPanel.getPlayers().size()).append(",");
-      text.append("\n");
-      text.append("Number of Alliances: ,");
-      text.append(statPanel.getAlliances().size()).append(",");
-      text.append("\n");
-      text.append("\n");
-      text.append("Turn Order: ,");
-      text.append("\n");
-      final List<PlayerId> playerOrderList = new ArrayList<>(gameData.getPlayerList().getPlayers());
-      playerOrderList.sort(new PlayerOrderComparator(gameData));
-      final Set<PlayerId> playerOrderSetNoDuplicates = new LinkedHashSet<>(playerOrderList);
-      for (final PlayerId currentPlayerId : playerOrderSetNoDuplicates) {
-        text.append(currentPlayerId.getName()).append(",");
+      writer.append(defaultFileName).append(',').append('\n');
+      writer.append("TripleA Engine Version: ,");
+      writer.append(ClientContext.engineVersion().toString()).append(',').append('\n');
+      writer.append("Game Name: ,");
+      writer.append(gameData.getGameName()).append(',').append('\n');
+      writer.append("Game Version: ,");
+      writer.append(gameData.getGameVersion().toString()).append(',').append('\n');
+      writer.append('\n');
+      writer.append("Current Round: ,");
+      writer.append(String.valueOf(currentRound)).append(',').append('\n');
```

Comment:
Note that using a `PrintWriter` would allow you to avoid the explicit `String` conversions you had to add in the previous commit.  You could also use the `println()` methods to avoid explicitly writing the (platform-dependent) newline everywhere.  However, `print()` and `println()` do not permit method chaining.

Another thought regarding the explicit newlines if you don't want to go the `PrintWriter` route...  You could change the type of `writer` to `BufferedWriter` and then replace the calls to `.append('\n')` with `.newLine()`.